src/lib/utils/hits-query-id.ts

```diff
@@ -1,6 +1,6 @@
 import { Hits } from '../../types';
 
-export const addQueryID = (hits: Hits, queryID: string) => {
+export const addQueryID = (hits: Hits, queryID: number): Hits => {
```

Comment:
Why was the `queryID` converted to a number?

Reply:
For what I understand it was an error since `Hit.__queryID` is typed as a `number` in [src/types/instantsearch.ts](https://github.com/algolia/instantsearch.js/blob/develop/src/types/instantsearch.ts#L45-L70) and `addQueryID` is returning an array of `Hit`.

Reply:
Let's see if @tkrugg can confirm.

Reply:
I tried it in the storybook and it is a string:

![image](https://user-images.githubusercontent.com/6137112/56643413-e45eba00-6679-11e9-92c3-05611894a229.png)

The type in `instantsearch.ts`  is therefore wrong.

Reply:
Ok, I fix it the other way around then :+1:

Reply:
yes `queryID` and `__queryID` should be of type string.